python/katana/_property_graph.pyx

```diff
@@ -140,13 +142,13 @@ cdef class PropertyGraphBase:
         Can be called from numba compiled code.
         """
         cdef uint64_t prev
+        #TODO(amber): need to dereference GraphTopology::edge_begin()/edge_end()
+        # here
```

Comment:
Why do you need to dereference them here? They are dereferenced below.

Reply:
That was a TODO for you :). You already took care of it. I'll go ahead and remove it.